### ISSUE TITLE
boost events with high response

### ISSUE THREAD

**Issue by jmsardain** [jmsardain/MLClusterCalibration#1]
events with a high response (typically are cluster with very low energy) have the highest chances to not be seen by the nn. 
Maybe we should look into how to pass these events more often. 

Even with a training of 500 epochs, and using a gaussian mixture (which turned out to be better than log normal), no migration to low energies has been seen, but it's still struggling to get good results for energies between 0.3 and 0.6 GeV. 

@MiLuchmann any ideas?

**Comment by ghost** [comment]
I could look into this problem. A simple enhancement of training in this region could maybe do the trick? For instance, by artificially enlarging the training data set by putting copies of data points into the region?

**Comment by jmsardain** [comment]
in that case, a suggestion to add to the reading of the dataset before the transformation: 

`mask = df['r_e_calculated'].gt(6)`
`df1 = df[mask].copy()`
`#repeat rows "response" times`
`df1 = df1.loc[df1.index.repeat(df1['r_e_calculated'].astype(int))]`

`#add rows, sorting and create default index`
`df = pd.concat([df1, df[~mask]]).sort_index().reset_index(drop=True)`

solution taken from [here](https://stackoverflow.com/questions/61833866/duplicate-rows-based-on-value-with-condition)

However, this should only be considered for the training dataset, the test dataset should remain intact.. This also means that the scales we will quote and save in the text file will actually be skewed. so we should also save the scales for the test dataset as well.. 
@MiLuchmann do you agree?
